### PR TITLE
feat: adding node12x builder

### PR DESCRIPTION
**Pull request by dennisnewel**
adding node 12 builder as requested by https://github.com/jenkins-x/jenkins-x-builders/issues/526

### CODE REVIEW

**Comment by garethjevans** [comment]
@dennisnewel you need to update the jenkins-x.yml to ensure these are built on release & pull-request

**Comment by dennisnewel** [comment]
ah, thanks! not as simple as i thought ;) pushing up changes shortly

**Comment by dennisnewel** [comment]
didn't prepare my local branches properly, so this PR effectively includes https://github.com/jenkins-x/jenkins-x-builders/pull/727

File: jenkins-x.yml [hunk 1]
```diff
@@ -111,6 +111,15 @@ pipelineConfig:
                     - --cache-repo=gcr.io/jenkinsxio/cache
                     - --cache=true
                     - --cache-dir=/workspace
+                - name: build-and-push-nodejs12x
```
Comment (abayer) [inline jenkins-x.yml]: This is failing due to the new dockerfile not being on this branch. =)

Comment (dennisnewel) [reply jenkins-x.yml]: gah...i really need to remember to reset my master before branching off again 🤦‍♂

**Comment by abayer** [comment]
So, you should edit `update-bot.sh` here to include the new builders - that won't make anything happen yet, but it has to be done eventually, might as well do it in this PR too.

Then, once that's done and this gets merged, you'll need to add the new builders to `jenkins-x-platform` (like https://github.com/jenkins-x/jenkins-x-platform/blob/master/jenkins-x-platform/values.yaml#L1124-L1179). Finally, you'll need to add yaml files for the new builders in https://github.com/jenkins-x/jenkins-x-versions/tree/master/docker/gcr.io/jenkinsxio. But don't do either of those things until this is merged. =)

**Pull request merged**